src/OVAL/oval_sysModel.c

```diff
@@ -411,6 +411,22 @@ xmlNode *oval_syschar_model_to_dom(struct oval_syschar_model * syschar_model, xm
 	return root_node;
 }
 
+struct oscap_source *oval_syschar_model_export_source(struct oval_syschar_model *model, const char *file)
+{
+	__attribute__nonnull__(model);
+
+	LIBXML_TEST_VERSION;
+
+	xmlDocPtr doc = xmlNewDoc(BAD_CAST "1.0");
+	if (doc == NULL) {
+		oscap_setxmlerr(xmlGetLastError());
+		return NULL;
+	}
+
+	oval_syschar_model_to_dom(model, doc, NULL, NULL, NULL);
+	return oscap_source_new_from_xmlDoc(doc, file);
+}
+
```

Comment:
This is very similar to `oval_syschar_model_export`. Basically duplicate.

Can you please make `oval_syschar_model_export` to call `oval_syschar_model_export_source`?